projects/demo-playwright/tests/addon-table/table-pagination/table-pagination.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, Page, test} from '@playwright/test';
+
+test.describe(`TablePagination`, () => {
+    test.describe(`Dropdown with \`[size]\`-options (amount items per page)`, () => {
+        test(`Basic case`, async ({page}) => {
+            await tuiGoto(page, `/components/table-pagination/API`);
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
+
+            await linesPerPageSelect.click();
+
+            await waitForCheckmarkIcon(page);
+            await expect(tablePagination).toHaveScreenshot(`0-[size]-dropdown-base.png`);
+        });
+
+        test(`With very long option name`, async ({page}) => {
+            const longNumber = Number.MAX_SAFE_INTEGER;
+
+            await tuiGoto(
+                page,
+                `/components/table-pagination/API?items=[0, ${longNumber}]&size=${longNumber}&total=${longNumber}&page=0`,
+            );
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
+
+            await linesPerPageSelect.click();
+
+            await waitForCheckmarkIcon(page);
+            await expect(tablePagination).toHaveScreenshot(`1-[size]-dropdown-base.png`);
```

Comment:
![1--size--dropdown-base-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/2061a7d8-0614-4dd4-b4e0-c0054c923310)

Reply:
(i think we can change width a little bit too)

Reply:
changed in 7edb545c0c79854fe9b037e5fd74e91c144444f8